mods/ra2/rules/allied-structures.yaml

```diff
@@ -397,12 +397,28 @@ gaweap:
 	WithIdleOverlay@bib:
 		Sequence: bib
 		RequiresCondition: !build-incomplete
+	GrantConditionOnProduction@helicopters:
+		Actors: shad
+		Condition: roof-open
+		Duration: 50
+		ShowSelectionBar: false
+	GrantConditionOnProduction@zeppelins:
+		Actors: zep
+		Condition: roof-open
+		Duration: 100
+		ShowSelectionBar: false
```

Comment:
This seems to be not working but it is an upstream issue.

Reply:
I see a pink selection bar during the roof opening.

Reply:
Yes, that's what the value is meant to remove.

Reply:
Fixed https://github.com/OpenRA/OpenRA/pull/17704.